docs/intro/schema.rst

```diff
@@ -226,7 +226,8 @@ understand backlink syntax is to split it into two parts:
 
 ``[is Movie]``
   This is a *type filter* that filters out all objects that aren't ``Movie``
-  objects.
+  objects. A backlink still works without this filter, but could contain any 
+  other number of objects besides `` Movie``s.
```

Comment:
```suggestion
  other number of objects besides `` Movie`` objects.
```

Sphinx doesn't seem to cooperate when you have inline code butted up against another character. I usually just write around it like this.

Reply:
Ah! Good to know.